### ISSUE TITLE
update on crowdflower elite?

### ISSUE THREAD

**Issue by nihlathar** [rainforestapp/tester-chrome-extension#568]
Hi. Because I have not received any answer from support, either CF or RFQA, and there isn't any other way to get in touch with them, I thought maybe here somebody will know something. As many of you probably know Crowdflower has stopped issuing withdrawals for workers receiving their payment through their own platform "elite", several weeks ago. The fact that I haven't seen any threads about this subject here perhaps suggests that not many people were affected, or maybe it's being discussed on the new/beta forum that few users were invited to. In any case, it's a huge problem. Was anyone able to find something out about this? When will they fix it? What the hell happened anyway that takes so long to solve? If you know anything please share it, thanks

**Comment by helloworld111gh** [comment]
Please see [this support article](https://communitysupport.crowdflower.com/customer/en/portal/articles/1319052-what-can-i-do-if-my-rewards-site-refuses-to-let-me-withdrawal-money-i-earned-from-crowdflower-tasks-) from Crowdflower:

**What can I do if my Rewards Site refuses to let me withdrawal money I earned from CrowdFlower Tasks?**

> Please contact the Contributor Support Team. 
> 
> We take payment issues very seriously and certainly want to make sure that Contributors get paid for the work they honestly completed. **Although CrowdFlower does not handle withdrawals for Contributors**, we do want to make sure that our partners do this properly. As a result, if you do encounter an Admin for your Rewards site that refuses to let you withdrawal, the first course of action is to immediately file a 'payment' issue ticket to the Contributor Support Team. Please provide all the detail you can -- amount, number of Tasks, etc. Also, if you have corresponded with the Admin for your Rewards site, please provide us with the details regarding why and how they have refused your withdrawal. 
> 
> We will then correspond with the Admin to better understand the issue. Because withdrawals are handled by the Admins of the various Rewards Sites the Support Team has limited options. That being said, CrowdFlower is selective as to who we accept as a Partner and our maintaining these relationships is largely dictated by your (Contributors) satisfaction. If you experience an on-going problem with one of CrowdFlower's Partners, please don't hesitate to contact the Community team for further investigation.  

However, as the site in question is literally just a generic EvolutionScript/PTCEvolution bux PTC script site, I would not give high hopes.

**Comment by nihlathar** [comment]
Friend, you obviously aren't informed about the issue. "Elite" is crowdflower's own site to collect payment, thereby avoiding the middle man that the ptc sites are. This way you actually get paid the full 4cents for every step completed, like it says on every hit you do. Anyway, all's been good until they stopped paying. There's a bunch of angry people on their facebook, but that doesn't seem to bother them much. It's been weeks at this point

**Comment by helloworld111gh** [comment]
You should still contact Crowdflower, as your rewards site (in this case
operated by Crowdflower itself) is currently refusing to pay you.
If after contacting Crowdflower and the rewards site admin you are unable
to get your payment you should assume that the site has decided to scam and
run with your money.
Remember that doing tasks on PTCEvo/EvoScript sites is risky, as there are
many instances of such operators scamming their users.

**Comment by nihlathar** [comment]
Normally I would tend to agree with you completely, but it's literally just them, it's not an external site operated by other people. Paypal money always came from accounting@crowdflower.com. So there's nobody else to contact. They just told me to wait, like everybody else, I know many other people have the same problem (that do other work beside Rainforest on CF). I just wanna know what's taking so long, when will we get paid.

**Comment by begsoon** [comment]
@helloworld111gh So,making money with Rainforest QA is also risky? Because money of testing go through CrowdFlower  to our PayPal( CrowdFlower Elite)

**Comment by nihlathar** [comment]
@begsoon you're in the same boat as me?

**Comment by uangonline** [comment]
@nihlathar Sorry to hear that you have payment issue with elite. I've been in your shoe before, so i know how it felt. If contacting CF support didn't resolve your problem, please contact "Kristen" as she willing to help. 

Please read https://portal.rainforestqa.com/ to see how to contact Kristen.

**Comment by nihlathar** [comment]
@uangonline are you getting paid through Elite at the moment and have no problems? 

Yup I wrote to rainforest too, as I mentioned, can't get an answer from them either (Im always polite, professional etc..). No clue what's happening

**Comment by uangonline** [comment]
@nihlathar Yes i got my payment yesterday.

**Comment by helloworld111gh** [comment]
@nihlathar If Crowdflower Elite refuses to pay you after about ~1 month, you should just cut your losses and stop using the site. Many sites go scam every week, and thus there is a possibility that you will not be paid for your time in the PTC/GPT industry.
You should also stop using the site if you are required to use a 3rd party to collect your money (aka you did not get it by contacting the rewards site or Crowdflower, and you had to use a Task Author/lawyer to get the funds). If you had to use a 3rd party to get payment, it is very likely it will be difficult/impossible to get paid again.
@begsoon It is only risky if you use a site that is very new, very cheaply made (ex. using pre-made script without customization, or using pirated script), or are using a site that selectively/does not pay.

**Comment by uangonline** [comment]
@helloworld111gh If Elite became scam, than CF also became scam. These 2 sites manage by the same company.

**Comment by nihlathar** [comment]
@uangonline well that's good to hear at least, Im thinking maybe the withdrawals made between last 2-3weeks are the "affected" ones, I dont know if you got paid every time during this time or not, maybe the next request I do will go through ok but it's so discouraging as I accumulated almost $100 in pending, can't afford to work for free anymore

@helloworld111gh I know the script they use is completely generic and old, but it is legit and it's just for requesting payment nothing else there's no contact info or anything there.

**Comment by uangonline** [comment]
@nihlathar Are you asking for payment more than once per week? I have no problem withdrawals my money last month either.

**Comment by RnkSts** [comment]
I Send various tickets to CF Support, First time they say that I "_have worked in a job [...]_" and they are reviewing my responses. 4 days after this, I reply the ticket again 2 times, and finally get this response.

>Hi,
>
>Very sorry for the Paypal payment issue. I have manually issued your payment and it should now be in your PayPal account. We appreciate your patience.
>
>Best, 
>
>CF

And finally i get my money in mi Paypal account (Requested 2 weeks ago).

I thing CF is watching more closely contributors, as there are a lot of people using VPN or Proxys to access the demographic jobs.

 CF Elite is far from being scam. But its very rare for something so important not to have an official statement from the CF Community Support apart from a comment in one of his post on Facebook.
![screenshot_1](https://cloud.githubusercontent.com/assets/20311303/18533921/b6daa0f2-7ab4-11e6-8bba-a8d6f11def78.png)

**Comment by JPierreS** [comment]
In regards to CF Elite some payments are on hold for the time being to investigate possible abuse with some tasks.  Yes it is sad that so many in the CF community are cheating and breaking rules, using VPN / Proxy when specifically prohibited from doing so and exploiting / cheating tasks.  The same issue is happening with RF.  It seems some people are keen in trying to destroy the crowd sourcing industry - Many of the top jobs are no longer present on CF due to abuse and still some people insist on abusing CF.

CF has a policy not to reverse funds to external channels, but of course as far as CF Elite (CF's own channel) it keeps a much closer eye - Sadly, as with anything in life, some honest contributors might be caught in the crossfire.  Despite the last batch of CF bans, people are still cheating and still using proxies to access some specific tasks they are not supposed to or submit fraudulent work.

On one side it is easy to blame CrowdFlower, but the abusive community is not helping matters.

I am sorry to hear that some of you are having issues with CF Elite, especially those who have done nothing wrong and not concerned by the cheat or abuse.  

Hopefully this will be resolved soon, but if they have to keep a close eye and check more submissions it only makes matters worse.  Don't open multiple tickets for the same issue, re-open or update your ongoing ticket and ask for an update after a reasonable time passed.

If you have not heard from CF after a reasonable delay, there are some legal venues you can take.

**Comment by MarinaCec** [comment]
@JPierreS very sad that good contributors pay for the mistakes of others... I don't even know how to use  proxy or vpn.

**Comment by JPierreS** [comment]
They are probably checking / investigating individually which can take time.  This is like anything else on CrowdFlower, such as giving flags for no reason or some users that got banned by mistake, etc.  Hopefully this will be resolved soon and people will get paid - If you are one of the good workers that has done nothing wrong try to keep up to date and ask about payment.  There is more scrutiny on CF Elite of course in regards to payments - there is a price to pay to get 0% deduction  - which is why I rather donate a % to a channel and get peace of mind and good support, where CF lacks severely sadly, and now they no longer have a community manager which they used for a year and let go, so I feel bad for those who are relying on the Rainforest task and STUCK on CF Elite.  Some users are being paid normally, some are owed 3 figures and have not done anything wrong.   Hopefully this can be sorted out soon.  If they have to check individual accounts and payments, it could take a lot of time but again this is the price to pay for the poor actions of others 
On the other hand it appears that with competitor crowd sites you get credited only after work was "approved" and checked and the process can be lengthy - so CF still remains "competitive" in terms of speed of payments

**Comment by MarinaCec** [comment]
I hope they solve this case with care, I think most contributors are honest, cheaters must be isolated cases.

It's a shame that whenever I reopen my ticket they never answer me or give clues to how the case goes.

**Comment by Genesis2015** [comment]
I have the same issue as you. Many of my friend too 👎

**Comment by JPierreS** [comment]
To those who are having the issues with CF Elite, how often are you requesting payment ? (every week ? every 2 weeks, etc.?) Are you working on RF task or other tasks as well ?

**Comment by nagabhyru** [comment]
I got last week and this week payments with out any problem. But I to did not get my previous payments which are equal to 80$. Now a days I am only working in RF and completely stopped working in CF.

**Comment by Genesis2015** [comment]
Absolutely the same story to me as @nihlathar mantioned above with only difference that I wating $180.

**Comment by MarinaCec** [comment]
> To those who are having the issues with CF Elite, how often are you requesting payment ? (every week ? every 2 weeks, etc.?) Are you working on RF task or other tasks as well ?

I work only on Rainforest and rarely 2 tasks that I like, I have requested payment every week

**Comment by uangonline** [comment]
@MarinaCec Are you also have payment pending in Elite?

**Comment by MarinaCec** [comment]
@uangonline yes, but in withdraw history it's as Completed, but I never received in my paypal

**Comment by sweeneyapps** [comment]
in my opinion, mturk is easier but I know it's only good for India and USA for cash.

**Comment by uangonline** [comment]
@MarinaCec Have you tried to contact them via Facebook and twitter? It seems contact them via this 2 channels will get faster respond.

@sweeneyapps Yes, beside India and USA, no one can withdraw their money from mturk to any bank account.

**Comment by nihlathar** [comment]
I noticed my ticket on CF's support got updated with yesterday's date stamp even though there was no new reply, hopefully it means they're making sure it stays open and doesn't expire, which at least gives some glimmer of hope

By the way did any of you guys try and message Rainforest directly like it tells us to do in the news on the portal? Did Kristen get back to any of you with some explanation? I can't get any answer -.-

**Comment by zfonck123** [comment]
im work with elite crowflower, for more one year. and not problems

**Comment by nagabhyru** [comment]
I to have a similar situation like @MarinaCec  I got my last two weeks payment. Rest of them did not. For me also there status is completed but I never receive them in my paypal account

**Comment by ladystoneheart** [comment]
I withdraw my money on Tuesday and that night after received my payment $300. nothing is wrong with them 
![untitled](https://cloud.githubusercontent.com/assets/20522240/18576276/c095626e-7be8-11e6-92a7-b7812ff69f32.png)

**Comment by abrahm12** [comment]
i am also waiting for payment since 3 weeks. No reply from crowdflower. I don't know what they are doing. Got reply from RF with same words they are investigating on some tasks.

**Comment by kristen-allie** [comment]
@nihlathar Last I heard from CF, they told me they were reviewing Elite accounts before starting payments. They are supposed to let me know when things are back to normal. I haven't heard back from them, so I emailed them again today. I will post a portal news update about it when I know more.

**Comment by JPierreS** [comment]
The process of reviewing all those accounts in question is lengthy.   CF has been facing a lot of issues, unfortunately, in the past in regards to contributors trying to taking advantage of the system, exploiting tasks, bypassing some limits such as geotargeting, multiple accounts, and fraudulent task submissions.
CF's policy is not to reverse funds from external channels, however it is much more careful with it's own channel (Elite) - this is what it comes to when a group of people abuse the system, everyone pays.

I think Rainforest should seriously consider doing the same and review accounts.  Now I know I am
going to get tons of thumbs down for this, it's ok, bring it on LOL !
I just care about this task, like many ;))

**Comment by mariaelite** [comment]
According to current international law. It is not legal withhold payments if illegality is not proven.
It is not the first time it happens with Elite.
The fact that no answer is a lack of respect and seriousness.
I just think swindled.
And they go 5 weeks without a single response.
When they happen to others, stop defending the indefensible ..
This behavior is not acceptable.
In fact, if further payments are received, it is proof that you were already investigated, then ... why they not pay what they owe?

**Comment by nihlathar** [comment]
@kristen-allie It is good to hear you're trying to make them accountable on our behalf. I think you have a better chance at getting some answers, than us.

@JPierreS I don't think people have a problem with the fact they're taking action against cheaters, it is the fact that your funds are being frozen for such a lengthy time without apparent reason, with no guarantee of ever getting them. For people that work this full part or part time, it means a large portion of their income just goes poof until a resolution comes along. Can you imagine your boss simply telling you won't get paid for several weeks, only because of he can't take the time right now to check whatever might be wrong with your past work, for example? That's unacceptable

@mariaelite afaik, there is just 1 admin designated to the task and he's really taking his time like it's no big deal that so many people are waiting.

**Comment by mariaelite** [comment]
nihlathar It is not acceptable excuses that can give.
5 weeks without a response from CF.
They are irresponsible and swindlers.

**Comment by nadamostafa24** [comment]
i too have the same issue 80 dollars pending and no response from CF support

**Comment by kristen-allie** [comment]
I received an update from CF:

```It looks like there are quite a few contributors. We are continuing to investigate this week. We are amassing a list of contributors to pay in a batch dispersal of funds. We underestimated how long it takes to review these individual accounts```

 So it looks like things are still being investigated, but some of you may start seeing your funds before others depending on what "batch dispersal of funds" you're in. I'll update you if I learn more!

**Comment by nihlathar** [comment]
That is good news, you put my mind at ease Kristen, thank you

**Comment by abrahm12** [comment]
I hope we will see some payments tomorrow. Thanks @kristen-allie and rainforest

**Comment by abrahm12** [comment]
Today my payment is released. Thanks to rainforest support and CF.

**Comment by HW71** [comment]
Just to make sure everyone impacted by this nasty CrowdFlower Elite issue gets the latest development / news on that:
Please check the information provided by @kristen-allie in the "News"-section of the Tester Portal at "2016-10-06 14:50:37 UTC" (you might need to scroll down a bit since additional news might be added) - and also make sure to consider the 24 hr deadline mentioned there:

https://portal.rainforestqa.com/

> Banned (Or Unpaid) CF Elite Users - Next Steps 2016-10-06 14:50:37 UTC, Kristen Hohenstein
> Another announcement for CF Elite users here! If you still never received your funds from the last month from CF Elite or were banned from CrowdFlower during this process without a clear explanation, please send in a support ticket categorized as "Other" with the first sentence "October CF Elite Issues" and your worker ID. **I'll give 24 hours for all of these tickets to come in**. I will then review your accounts, and if your job history shows you are a good/trustworthy tester, I will email CF on your behalf to see if we can get the ban removed. Ps. If you have already sent me a message on facebook or the forum, please send me your ID again following the above procedure. I apologize for the repetitiveness -- this will just ensure no IDs slip through the cracks <3
>

**Comment by luisdanielvc** [comment]
What can we do now that the 24 hours passed? A friend just discovered that he hasn't been paid by elite in 3 weeks

**Comment by doncoi151189** [comment]
After "CF elite" pay me in before in October, more than five weeks of waiting, continue after October, I continue to wait more than two weeks. Despite submit a support ticket, but I still do not get any assistance from the CF, this year, I always have to submit weekly support ticket to getting paid.
ID CF elite: 35085118

**Comment by jaimes20031** [comment]
to me they have not paid me in elite, my payment was in pending since August and says he was sent but never arrived, is $ 94.04 will even send the ticket to kristen and still waiting to answer me, and spent 24 hours, I it hurts to lose those $ 94 because of CrowdFlower as were more than 15 days of work in rainforest

**Comment by nihlathar** [comment]
@jaimes20031 I waited for exactly 1 month, they will get to you eventually and you will get paid if you didn't break their rules (somehow must have gotten in the net, I for example used vpn, which is forbidden, even though I didnt create new acounts or anything of that nature just used it for other things, but was forgiven). Be patient Kristen should respond to you, you could have sent a support ticket on 2 occasions that the memo was posted in rainforest portal because she was collecting IDs to send to crowdflower. You could try to reply to your own support ticket on crowdflower, or create a new one if you haven't already but don't spam them too much.

**Comment by thaodonkey** [comment]
I withdraw from elite crowflower account, in my account the payment status is received but i checked in paypal no have money. I withdraw one week ago and no response form crowflower, anyone have same problem with me?

**Comment by HW71** [comment]
Hi @thaodonkey,

since we now have this great discussion forum over there at

http://forum.rainforestqa.com/

we should avoid general discussions not related to the Chrome extension here in this GitHub issue tracker. 

IIRC there has already been a thread opened for this CrowdFlower Elite issue => just give it a try...

Have a great one & Cheers, Erik
